src/fetch/index.ts

```
@@ -21,19 +21,32 @@ import { UrlPrefixPattern, ToUrlParamPattern } from "../core";
+    ? [QueryKeys] extends [never]
+      ? true
+      : false
+    : false,
+> = [QueryRequiredError] extends [true]
+  ? QueryParameterRequiredError
+  : // If Url is given as like "https://example.com?${string}", then QueryKeys will be string
+    string extends QueryKeys
+    ? C.OK
+    : ValidateQuery<
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/ban-types
+        QueryDef extends Record<string, any> ? QueryDef : {},
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)